scripts/custom-screens/CustomFormControls.tsx

```diff
@@ -22,7 +22,8 @@ import {
   DropdownField,
```

Comment:
When making changes to app generation always do `npm run bootstrap-react-app` before commit. `example-react-app` is not updated with these changes.